File: Examples/Getting-Started/Get-Device-Hardware/nodeJS/get-device-hardware.mjs
import "dotenv/config";

import { NetLDClient, printDeviceHardware } from "./netld-example-client.mjs";

const client = NetLDClient.fromEnv();
await client.login();

const hardware = await client.getDeviceHardware({
  ipAddress: process.env.NETLD_DEVICE_IP || "10.95.1.40",
  network: process.env.NETLD_NETWORK || "Default",
});

printDeviceHardware(hardware);

File: Examples/Getting-Started/Get-Device-Hardware/Python/get_device_hardware.py
import os

from netld_example_client import NetLDClient, print_device_hardware


client = NetLDClient.from_env()
client.login()

hardware = client.get_device_hardware(
    ip_address=os.environ.get("NETLD_DEVICE_IP", "10.95.1.40"),
    network=os.environ.get("NETLD_NETWORK", "Default"),
)

print_device_hardware(hardware)
